portfolio/src/main/webapp/commentScript.js

```diff
@@ -29,17 +43,48 @@ function handleResponse(response){
 
 /** Adds user and reflection to html page **/
 function addCommentsToDom(commentList){
-    const commentContainer = document.getElementById('comments-container');
+    const commentContainer = document.getElementById(CONTAINER_SERVLET);
     commentContainer.innerHTML = '';
-    for (let i = 0; i < commentList.length; i++){
+    for (let i = 0; i < commentList.length; i++) {
         commentContainer.appendChild(
-            createComment(commentList[i].username, commentList[i].reflection, commentList[i].timeStamp));
+            createComment(commentList[i]));
     }
 }
 
-/** Creates an <li> element containing author: comment. */
-function createComment(user, reflection, timeStamp) {
-  const liElement = document.createElement('li');
-  liElement.innerText = user + "--> " + reflection + "\n" + timeStamp + "\n\n\n";
+/** Creates an <li> element containing comments. */
+function createComment(comment) {
```

Comment:
At this point, I think this method name should be something like `commentsContainer`